saleor/channel/migrations/0004_create_default_channel.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+def create_default_channel(apps, schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+
+    Channel.objects.create(
```

Comment:
We already have a migration "channel/0001" which creates a default channel in some cases; it also happens to use "default-channel" as a slug. If such an instance is already in the database, this migration will fail as slug needs to be unique. We should use `get_or_create` in this migration to avoid it. Also since we have `DEFAULT_CHANNEL_SLUG` env, we could use it here as well.